docs/news.rst

```diff
@@ -2,6 +2,68 @@
 Release Notes
 =============
 
+2.5.0 (2021-10-05)
+==================
+
+Highlights
+----------
+
+- Next major release (3.0) will remove all non-standard single dash long parameter cli parameter
+
+- Added ability generate manifest file from local directory structure
+
+  .. code-block:: bash
+
+        # from the command line
+        synapse manifest ./ --parentId syn123 --manifestFile ./manifest.tsv
+
+- Added ability to generate manifest file from local directory structure
+
+  .. code-block:: bash
+
+        # from the command line
+        synapse manifest ./ --parentId syn123 --manifestFile ./manifest.tsv
```

Comment:
@BrunoGrandePhD  What do you think creating a default for the `--manifestFile` parameter?  Also should we rename it `--manifest-file` as we are slowly trying to move away from camel case?  I can make these changes, just wanted your thoughts.

Reply:
I'm happy with renaming the argument name to kebab or snake case. As for a default value, I would like it to output to stdout by default. I can do this later this week once I get a high-priority task off of my plate, but I'm fine if you make the update.